src/eei.cpp

```diff
@@ -744,6 +742,17 @@ inline int64_t maxCallGas(int64_t gas) {
 
       call_message.gas = gas;
 
+      if (import->base == Name("call") || import->base == Name("callCode")) {
```

Comment:
Does this needs to be moved from the original position?

Reply:
Not sure what you mean by "moved." The original still exists at line 673. This line was added because this change only applies to `call` and `callCode`, not to the other host functions handled in this function.

Reply:
To me it seems the original line 686 was removed and was moved over here?

Reply:
I see what you mean. I misread which line you were referring to. Fair point. Let me move and test.

Reply:
I tested with the check in the original location. The test fails because no gas is consumed. We need to charge some gas in this case. See e.g. the logic at https://github.com/ethereum/py-evm/blob/master/evm/vm/logic/call.py#L65.

Reply:
I'm not sure all this random logic matches with the YP, so frustrating.